bridge/Cargo.toml

```diff
@@ -3,28 +3,29 @@ name = "bridge"
 version = "0.2.0"
 
 [dependencies]
+error-chain = "0.11.0-rc.2"
+ethabi = "5.1"
+ethabi-contract = "5.0"
+ethabi-derive = "5.0"
+ethcore = { git = "http://github.com/paritytech/parity", rev = "991f0ca" }
+ethcore-transaction = { git = "http://github.com/paritytech/parity", rev = "991f0ca" }
+ethereum-types = "0.3"
 futures = "0.1"
+itertools = "0.7"
+jsonrpc-core = "8.0"
+keccak-hash = { git = "http://github.com/paritytech/parity", rev = "991f0ca" }
+log = "0.3"
+pretty_assertions = "0.2.1"
+reqwest = "0.8.5"
```

Comment:
Word of caution: as far as I remember, `reqwest` brings a significant number of dependencies. Some of them blend with our existing dependencies, however. Not a huge deal.

Reply:
I am going to try to change this to hyper.